theblues/plans.py

```diff
@@ -15,7 +15,13 @@
 
 Plan = namedtuple('Plan',
                   ['url', 'plan', 'created_on', 'description', 'price'])
-PLAN_VERSION = 'v2'
+Wallet = namedtuple('Wallet',
```

Comment:
Why two separate indentation styles for the Wallet and WalletTotal tuple types? [shrug]
I have a personal preference on the WalletTotal style, but I'd say let's keep them consistent.

Reply:
Side effect of a refactor. Fixed.